### PR TITLE
Fix frontend

### PR DESCRIPTION
**Pull request by at-ngatran**
Hi A Dung, I fixed frontend, please check it, thanks so much

### CODE REVIEW

File: resources/views/frontend/auth/login.blade.php [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    <div class="row">
+        <div class="col-md-8 col-md-offset-2">
+            <div class="panel panel-default">
+                <div class="panel-heading">Login</div>
+                <div class="panel-body">
+                    <form class="form-horizontal" role="form" method="POST" action="{{ url('/login') }}">
+                        {{ csrf_field() }}
+
+                        <div class="form-group{{ $errors->has('email') ? ' has-error' : '' }}">
+                            <label for="email" class="col-md-4 control-label">E-Mail Address</label>
```
Comment (dzungnguyenh) [inline resources/views/frontend/auth/login.blade.php]: [rank C] Why doesn't this label on language file?

File: resources/views/frontend/auth/login.blade.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                                    </span>
+                                @endif
+                            </div>
+                        </div>
+
+                        <div class="form-group">
+                            <div class="col-md-6 col-md-offset-4">
+                                <div class="checkbox">
+                                    <label>
+                                        <input type="checkbox" name="remember"> Remember Me
```
Comment (dzungnguyenh) [inline resources/views/frontend/auth/login.blade.php]: [rank C] Why doesn't this label on language file?

File: resources/views/frontend/layouts/partials/menu.blade.php [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+                  mouseport: $('body')}
+                                                          );
+              }
+               );
+            }
+            )(jQuery);
+          </script>
+          <div class="parallax-viewport" id="parallax">
+            <!-- parallax layers -->
+            <div class="parallax-layer" style="width: 1000px; height: 192px;">
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/menu.blade.php]: [rank A] please avoid using css inline and don't hardcode images path.

File: resources/views/frontend/layouts/partials/sitemaptop.blade.php [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<div class="siteMapTop">
+  <div class="siteMapTop_container">
+    <div class="fright siteMapTop_logout2">
+      <a href="/" onClick="accessToLogin(this); return false;" class="siteMapTop_color">
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/sitemaptop.blade.php]: [rank A] please don't use js like that. We need define it as jQuery click function.

**Comment by at-ngatran** [comment]
Hi a Dung, I fix mistake, please check it, thanks so much

File: resources/views/frontend/layouts/partials/colnav.blade.php [hunk 5]
```diff
@@ -0,0 +1,47 @@
+<div class="contentwrap">
+  <div class="colnav">    
+    <div id="firstlogin" class="box box_login ">
+      <div class="bg_t">
+      </div>
+      <div class="content">
+        <img src="frontend/imgs/quatang.jpg" alt="">
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/colnav.blade.php]: [rank C] why are you alway using hardcode paths? Please check and make it as Laravel path function (url ())

File: resources/views/frontend/layouts/partials/colnav.blade.php [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+      </div>
+      <div class="bg_b">
+      </div>
+      <div class="clear">
+      </div>
+    </div>
+    <div id="afterlogin" class="box box_login " style="margin-top:-20px;;">
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/colnav.blade.php]: [rank A] again. You shouldn't use css inline.

File: resources/views/frontend/layouts/partials/footer.blade.php [hunk 7]
```diff
@@ -0,0 +1,12 @@
+<div id="footer" class="footterwrap">
+</div>
+<!-- #Footer -->
+<div class="footer-seo" style="z-index: 20;">
+  <div style="width:980px;margin:0 auto;position: relative;z-index: 15;">
+    <div style="position:absolute;top:-20px;left:10px;">
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/footer.blade.php]: [rank A] again. You shouldn't use css inline.

File: resources/views/frontend/layouts/partials/footer.blade.php [hunk 8]
```diff
@@ -0,0 +1,12 @@
+<div id="footer" class="footterwrap">
+</div>
+<!-- #Footer -->
+<div class="footer-seo" style="z-index: 20;">
+  <div style="width:980px;margin:0 auto;position: relative;z-index: 15;">
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/footer.blade.php]: [rank A] again. You shouldn't use css inline.

File: resources/views/frontend/layouts/partials/menu.blade.php [hunk 9]
```diff
@@ -0,0 +1,155 @@
...
+                                                          );
+              }
+               );
+            }
+            )(jQuery);
+          </script>
+          <div class="parallax-viewport" id="parallax">
+            <!-- parallax layers -->
+            <div class="parallax-layer" class="layer1" >
+              <img src="{{asset('{{asset('frontend/imgs/hd_bg.jpg')}}" alt=""  id="img1" style=";" />
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/menu.blade.php]: [rank A] will you using id attribute for selector? Please avoid named it clearly more

File: resources/views/frontend/layouts/partials/menu.blade.php [hunk 10]
```diff
@@ -0,0 +1,155 @@
...
+            </div>
+            <div class="parallax-layer" class="layer2" >
+              <img src="{{asset('frontend/imgs/hd_may1.png')}}" alt="" id="img2"  />
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/menu.blade.php]: [rank A] will you using id attribute for selector? Please avoid named it clearly more
please don't use any variable or attribute name like abc1, abc2, abc3, ....

File: resources/views/frontend/layouts/partials/menu.blade.php [hunk 11]
```diff
@@ -0,0 +1,155 @@
...
+        <div class="hd_content">
+          <a class="logo" href="">
+          </a>
+        </div>
+      </div>
+      <div id="main-menu">
+        <div class="nav_bar">
+          <div class="content" style="float: none;">
+            <ul id="menu" class="nav">
+              <li class="trangchu trangchu2" style="display: block; position: relative;">
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/menu.blade.php]: [rank A] Again. You are using css inline. Please pay attention for these mistake, you are getting it so many times.

File: resources/views/frontend/layouts/partials/menu.blade.php [hunk 12]
```diff
@@ -0,0 +1,155 @@
...
+              </li>
+              <li class="tintuc">
+                <div class="content">
+                  <table>
+                    <tbody>
+                      <tr>
+                        <th>
+                          <a id="tab_vichamhoc" href="/#"" class="first-child hover_img">
+                            <img  src="{{asset('frontend/imgs/2_icon_vichamhoc.png')}}" class="img_icon"
+                                 style="width: 97px; height: 75px; left: 17px;" />
```
Comment (dzungnguyenh) [inline resources/views/frontend/layouts/partials/menu.blade.php]: [rank A] what's the matter with you when try to define css class?

**Pull request closed without merging**